themes/scipy_lectures/static/nature.css_t

```diff
@@ -601,11 +601,12 @@ div.body img {
 }
 
 /* Center figures that are larger than 100% of the figure */
-div.body div.figure {
+/* div.body div.figure {
     margin-left: -200px;
     margin-right: -200px;
     text-align: center;
-}
+  }
+*/
```

Comment:
@gertingold : could you tell me where this was in conflict with sphinx-gallery: which specific element of sphinx-gallery it cause to be a problem.